src/components/Gallery/index.tsx

```diff
@@ -456,12 +456,6 @@ const Gallery: React.FC<GalleryProps> = ({
             primaryFont={controlsFont}
             dangerouslySetInnerHTML={ImageCountTextOutput}
           />
-          <ControlsButton type="button" aria-label={previousImagePhrase} onClick={(): void => prevHandler()}>
```

Comment:
We are removing these completely from all viewports? If I am not mistaken the lighthouse audit would only fail these on mobile viewports? We could look at not showing these for mobile?

Reply:
yeah can do. I guess I was thinking of cleaning up and got overzealous

Reply:
updated @matthewroach